src/components/loading-spinner.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { ActivityIndicator, StyleSheet, Text, View } from "react-native";
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: "center",
+    flexDirection: "row",
+    padding: 10
```

Comment:
I don't think it's good to have this kind of styling on a loading component. When the loading finishes the component will resize, which I think can become a problem. What do you think @AdamJonsson?

Maybe like this instead?
```suggestion
    justifyContent: "center",
    flexDirection: "row",
```